test/verify/check-metrics

```diff
@@ -839,65 +839,89 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
         # No matching type
-        m.execute("mkdir -p /tmp/hwmon/hwmon0")
-        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
-        m.execute("mount -o bind /tmp /sys/class")
+        m.execute("mkdir -p /tmp/sensor-sys-class/hwmon/hwmon0")
+        self.addCleanup(m.execute, "rm -rf /tmp/sensor-sys-class/hwmon")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/name", "BAT0")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/temp1_input", "40000")
+        m.execute("mount -o bind /tmp/sensor-sys-class /sys/class")
+        self.addCleanup(m.execute, "umount /sys/class")
 
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
-        # create files that contain CPU temperature
-        m.execute("mkdir -p /tmp/hwmon/hwmon1")
-
-        # Testing on ARM devices
-        m.execute("echo cpu_thermal > /tmp/hwmon/hwmon1/name")
-        m.execute("echo 60000 > /tmp/hwmon/hwmon1/temp1_input")
-
-        b.wait_in_text("#current-metrics-card-cpu", "60 °C")
+        def clean_and_relog():
+            m.execute("rm -rf /sys/class/hwmon/hwmon1/*")
```

Comment:
In your test you only ever check hwmon1/, not a second one. This should be done, but can be a follow-up.

Reply:
I added a test where it checks hwmon1 and hwmon2 at the same time and shows the hottest sensor.